backends/openvino/requirements.txt

```diff
@@ -0,0 +1,9 @@
+datasets
+huggingface-hub
+safetensors
+sentencepiece
+tokenizers
```

Comment:
which of these requirements need to be in this folder?

Reply:
Updated the requirements to include only necessary packages for this PR.